internal/wire/wire.go

```diff
@@ -164,6 +164,7 @@ func viewPathToGraphPath(v *view.View, key string, graphDB *cayley.Handle) (*pat
 
 	// Loop over the path segments translating the path.
 	var graphPath *path.Path
+	var subPaths []path.Path
 	level := 1
 	for _, segment := range v.Path {
```

Comment:
This is silly stuff. These variable declarations should be separated from the for loop. They are really two separate concerns in the code.